client/src/components/TaskCard/TaskCard.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { PropTypes } from "prop-types";
+import SmallActionButton from "../SmallActionButton/SmallActionButton.js";
+import styles from "./TaskCard.module.css";
+
+const TaskCard = ({
+  title,
+  locationText,
+  startTimeText,
+  endTimeText,
+  isOngoing,
+  showDetailsDialog,
+}) => {
+  const [_isOngoing, setOngoing] = useState(isOngoing);
```

Comment:
I think we can remove the underscore and keep it just as `isOngoing`

Reply:
Renamed `_isOngoing` to `taskOngoing`. This was done to prevent confusion with the prop `isOngoing`.